### ISSUE TITLE
[Wise] Editing loss = nan + 0

### ISSUE THREAD

**Issue by YuJu0819** [zjunlp/EasyEdit#335]
I run the sample code of WISE and use the same ZSRE dataset file. It turns out that the loss is nan and the edit fail. Is there anything I misunderstand or I skip some step?

This is the code I use,
```

import sys
import os
import json
# Add the parent directory to the sys.path
from EasyEdit.easyeditor import WISEHyperParams, BaseEditor
from huggingface_hub import login


K = 1000
edit_data = json.load(open('./data/zsre_mend_edit.json', 'r', encoding='utf-8'))[:K]
loc_data = json.load(open('./data/zsre_mend_train.json', 'r', encoding='utf-8'))[:K]
loc_prompts = [edit_data_['loc'] + ' ' + edit_data_['loc_ans'] for edit_data_ in loc_data]

prompts = [edit_data_['src'] for edit_data_ in edit_data]
rephrase_prompts = [edit_data_['rephrase'] for edit_data_ in edit_data]
target_new = [edit_data_['alt'] for edit_data_ in edit_data]
locality_prompts = [edit_data_['loc'] for edit_data_ in edit_data]
locality_ans = [edit_data_['loc_ans'] for edit_data_ in edit_data]
locality_inputs = {
    'neighborhood':{
        'prompt': locality_prompts,
        'ground_truth': locality_ans
    },
}
hparams = WISEHyperParams.from_hparams('./EasyEdit/hparams/WISE/llama-7b.yaml')

hparams.model_name = "TheBloke/Llama-2-7B-fp16"
editor = BaseEditor.from_hparams(hparams)
metrics, edited_model, _ = editor.edit(
    prompts=prompts,
    rephrase_prompts=rephrase_prompts,
    target_new=target_new,
    loc_prompts=loc_prompts,
    locality_inputs=locality_inputs,
    sequential_edit=True
)
# a = 

```

**Comment by pengzju** [comment]
It's possible that the issue you're encountering with the loss becoming `nan` is due to **`fp16` overflow**. Have you experimented with editing "TheBloke/Llama-2-7B-fp16" using any other editing methods? Do you have the GPU resources to try using `fp32` instead?

**Comment by zxlzr** [comment]
hi, have you solved your issue yet?
